### ISSUE TITLE
Split <info> into multiple lines

### ISSUE THREAD

**Issue by szipper** [ropensci/hydrology#22]
From Achim:

> Currently, the `<info>` section is collapsed into a single line. This
> makes it very hard for me to edit it. I typically do this when packages on
> CRAN get archived so that I omit them from the .ctv. When you do the next
> update on GitHub you need to reflect this as well...and it also gives you
> the chance to review whether plain ommission is really the best strategy.
> Sometimes packages re-appear or are still on GitHub or there are
> replacement packages etc.

**Comment by boshek** [comment]
@szipper @ilapros Thoughts on what 
>plain ommission 

means?

**Comment by szipper** [comment]
Hmmm I don't know but it seems like it is something related to packages that are removed from CRAN. Maybe 'plain omission' is just deleting it from the <info> section (sounds like this is what Achim normally does)? This would be in contrast to the alternate approach, which would be changing the link to a GitHub package or something like that.

Just a guess though

**Comment by boshek** [comment]
In the interest of doing this quickly I ended up doing this manual by formatting the html code outside of R. Ideally we'd be able to use some command line tool.

**Comment by ilapros** [comment]
So given I am adding the package for #25 now  - should I simply edit the .ctv file and only compile the ctv --> html file. I think given the first big bulk of the work is done we can live with this workflow - but we would also need to update the contributing.md file?

**Comment by boshek** [comment]
@ilapros I think we should maintain the workflow we have because it also generates the README which I think is useful for people.

**Comment by ilapros** [comment]
So if I add the package to the hydrology.md I would then need to subsequently spit the .ctv manually?

**Comment by boshek** [comment]
Yes. But that is only because I haven't figured out a shell or R solution to formatting the .ctv info section that can be added to the makefile. So ultimately it would be automated but for now that info step is manual. FWIW I wouldn't worry about it as I have to add another package so you could skip that process for #25.

**Comment by ilapros** [comment]
Looking into whether stringr can do this right now - if I can hack it I'll share the code. That seems like a possible tool for the job

**Comment by ilapros** [comment]
I believe/hope that b14709baa52763e5e6a3332ca53d1c17ada5f018 closes this - I'd appreciate if you could give it a spin?

**Comment by boshek** [comment]
@ilapros this looks great! Well done. Will test it out later and report back. Great work.
